packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReviewQuery.js

```diff
@@ -0,0 +1,40 @@
+import gql from 'graphql-tag';
```

Comment:
This file should be named `itemsReview.gql.js` per the convention we discussed in slack. I just added it today in the wiki :)